content/applications/inventory_and_mrp/inventory/barcode/operations/adjustments.rst

```diff
@@ -1,48 +1,153 @@
-takes place.
+.. seealso::
+   :doc:`../../management/inventory_adjustments/count_products`
 
-.. image:: adjustments/adjustments_02.png
-    :align: center
+Enable Barcode app
+==================
 
-If you don’t work with multi-location, you will be able to scan the
-different products you want to include in the inventory adjustment.
+To use the *Barcode* app to create and apply inventory adjustments, it **must** be installed by
+enabling the feature from the settings of the *Inventory* app.
 
-.. image:: adjustments/adjustments_03.png
-    :align: center
+To do so, go to the :menuselection:`Inventory app --> Configuration --> Settings`. Then, scroll down
+to the :guilabel:`Barcode` section, and click the checkbox next to the :guilabel:`Barcode Scanner`
+feature.
 
-.. note::
-         If you have 5 identical articles, scan it 5 times or use the keyboard to
-         set the quantity.
+Once the checkbox is ticked, click :guilabel:`Save` at the top of the page to save changes.
 
-Besides using the barcode scanner, you can also manually add a product
-if necessary. To do so, click on *Add Product* and fill the
-information in manually.
+Once the page has refreshed, new options are displayed under the :guilabel:`Barcode Scanner`
+feature: :guilabel:`Barcode Nomenclature` (with a corresponding drop-down menu), where either
+:guilabel:`Default Nomenclature` or :guilabel:`Default GS1 Nomenclature` can be selected. The
+selected nomenclature changes how scanners interpret barcodes in Odoo.
 
-.. image:: adjustments/adjustments_04.png
-    :align: center
+There is also a :guilabel:`Configure Product Barcodes` internal link arrow, along with a set of
+:guilabel:`Print` buttons for printing barcode commands and a barcode demo sheet.
 
-.. image:: adjustments/adjustments_05.png
-    :align: center
+.. image:: adjustments/adjustments-barcode-setting.png
+   :align: center
+   :alt: Enabled Barcode feature in Inventory app settings.
 
-When you have scanned all the items of the location, validate the
-inventory manually or scan the *Validate* barcode.
+For more on setting up and configuring the :guilabel:`Barcode` app, refer to the :doc:`Set up your
+barcode scanner <../setup/hardware>` and :doc:`Activate the Barcodes in Odoo
+<../setup/software>`.
 
-.. image:: adjustments/adjustments_06.png
-    :align: center
+Create and apply inventory adjustment
+=====================================
+
+To scan product barcodes and apply inventory adjustments, navigate to the :menuselection:`Barcode
+app`.
+
+Once inside the *Barcode* app, a :guilabel:`Barcode Scanning` screen displaying different options is
+presented.
+
+To create and apply inventory adjustments, click on the :guilabel:`Inventory Adjustments` button at
+the bottom of the screen.
+
+Doing so creates a new adjustment, and navigates to the :guilabel:`Barcode Inventory Client Action`
+page, with :guilabel:`Inventory Adjustment` at the top of the form.
```

Comment:
I don't think the action of clicking the **Inventory Adjustments (#)** button actually creates a new adjustment, right? All it seems to do is direct to the new dashboard where adjustments can be created...

As well, does **Barcode Inventory Client Action** get a gui?

Here's a suggested rewrite:

```suggestion
Doing so navigates to the *Barcode Inventory Client Action* page , labeled as :guilabel:`Inventory Adjustment` in the top header section.
```